### PR TITLE
Add metrics exporter and documentation

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
We now provide a dedicated metrics exporter with CRI-O. Documentation has been updated as well.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
- Added metrics-exporter and [documentation](https://github.com/cri-o/cri-o/master/tutorials/metrics.md#exporting-metrics-via-prometheus)
```

### CODE REVIEW

**Comment by haircommander** [comment]
LGTM, though your release-notes are linking the openSUSE version of the commit, not the cri-o/cri-o version... We may want to point to `https://github.com/cri-o/cri-o/master/tutorials/metrics.md#exporting-metrics-via-prometheus` (which will 404 now, but be populated later, and show the latest up to date instructions)

**Comment by saschagrunert** [comment]
> LGTM, though your release-notes are linking the openSUSE version of the commit

Sounds good, I'll change that.

**Comment by saschagrunert** [comment]
Fixed the vendor and lint issues

File: tutorials/metrics.md [hunk 1]
```diff
@@ -49,3 +49,126 @@ Beside the [default golang based metrics][2], CRI-O provides the following addit
...
 [3]: https://github.com/kubernetes/cri-api/blob/a6f63f369f6d50e9d0886f2eda63d585fbd1ab6a/pkg/apis/runtime/v1alpha2/api.proto#L34-L128
+
+## Exporting Metrics via Prometheus
+
+The CRI-O metrics exporter can be used to provide a cluster wide scraping
+endpoint for Prometheus. It is possible to either build the container image
+manually via `make metrics-exporter` or directly consume the [available image on
+quay.io][4].
+
+[4]: https://quay.io/repository/crio/metrics-exporter
```
Comment (TomSweeneyRedHat) [inline tutorials/metrics.md:60]: Not at all related to the mergeability of this PR, but there's no "Description" text on the quay.io [repo](https://quay.io/repository/crio/metrics-exporter?tab=info) and there should be.

**Comment by TomSweeneyRedHat** [comment]
LGTM

File: tutorials/metrics.md (hunk 1 above)
Comment (saschagrunert) [reply tutorials/metrics.md:60]: Yep, I added a description which links to the documentation as well.

**Pull request merged**